proposals/stage-1/SUPPORTER_PROGRAM/README.md

```diff
@@ -0,0 +1,48 @@
+# Individual Membership Representation Proposal
+>  Stage 1
+
+## Champion
+
+[Sara Chipps](https://github.com/sarajo)
+ 
+## Description
+
+Membership program for OpenJS individual supporters. 
+
+## Required Resources
+
+What resources would be needed to execute it?
+* The current weekly newsletter signup is removed from the OpenJS website. 
```

Comment:
I think the newsletter should stay open to all project collaborators and members.

Reply:
That makes sense. Is there another offering we can give supporters that will incentivise people to join?

Reply:
Is this the same newsletter as the one we use for projects? My guess is this is a separate marketing based one.

If we offer free supporter status to contributors that could be a path for them to gain acess.

Are we going to stop supporting a public mailing list altogether here? Seems like that could be a step backwards for some marketing efforts such as advertising the conference.

Reply:
My understanding is that right now it has a little over a hundred subscribers. The suggestion was to use the newsletter that Jory send out as a supporter newsletter. 

If we're not in agreement about that then would love to brainstorm a new newsletter (and maybe figure out who would write/distribute it)? Or, something else we could offer that would incentivise people to sign up?

Reply:
TBH I don't have particularly strong feelings about this. We also have twitter and other social channels to send out comms... just want to ensure that we are not losing outreach channels

Reply:
We discussed this at the CPC meeting, it will stay open AND be a perk.